core/include/detray/tracks/free_track_parameters.hpp

```diff
@@ -28,47 +27,42 @@ struct free_track_parameters {
     using transform3_type = dtransform3D<algebra_t>;
     using matrix_operator = dmatrix_operator<algebra_t>;
 
-    // Shorthand vector/matrix types related to free track parameters.
+    // Shorthand vector type related to free track parameters.
     using vector_type = free_vector<algebra_t>;
 
-    // Track helper
-    using track_helper = detail::track_helper<matrix_operator>;
-
     /// @}
 
-    DETRAY_HOST_DEVICE
-    free_track_parameters()
-        : m_vector(matrix_operator().template zero<e_free_size, 1>()){};
+    /// Default constructor
+    free_track_parameters() = default;
```

Comment:
or we can name this to `free_vector()` and never use free_track_parameter. That will harmonizer better with the next PR

Reply:
Yes, absolutely

Reply:
I renamed the class/transform function (```free_track_parameters```->```free_parameters_vector``` and ```bound_to_free_param```->```bound_to_free_vector```), but I was not so sure about completely removing the name ```free_track_parameters``` as the symmetric type to ```bound_track_parameters```. So I added an alias, which makes clear, what free track parameters are:
```
/// The free track parameters consist only of the parameter vector
template <typename algebra_t>
using free_track_parameters = free_parameters_vector<algebra_t>;
```
So, now it is completely symmetric in its implementation to the bound track parameters and just not adding the covariance etc. It will also cause less changes in downstream code.

Does that work for you?